data/projects.js

```diff
@@ -447,6 +447,24 @@ coala should support generating metrics for your code.",
 		"collaborating_projects" : ["coala", "jenkins"]
 	},
 	{
+		"name" : "Build an ideas website for GSoC",
+		"desc" : "There are a number of possible ways to improve the \
+                          implementation of GSoC keeping [projects](https://projects.coala.io) \
+                          in mind.",
+		"requirements" : [
+			"One patch to projects accepted.",
+			"Clear ideas on how to improve the GSoC ideas implementation \
+                         in the proposal."
+		],
+		"difficulty" : "medium",
```

Comment:
Although I am lacking experience with web dev stuff I feel like this deserves an easy compared with our other projects